proto/zitadel/webkey/v2beta/webkey_service.proto

```diff
@@ -1,25 +1,23 @@
 syntax = "proto3";
 
-package zitadel.resources.webkey.v3alpha;
+package zitadel.webkey.v2beta;
 
 import "google/api/annotations.proto";
 import "google/api/field_behavior.proto";
+import "google/protobuf/timestamp.proto";
 import "protoc-gen-openapiv2/options/annotations.proto";
 import "validate/validate.proto";
-
 import "zitadel/protoc_gen_zitadel/v2/options.proto";
+import "zitadel/webkey/v2beta/key.proto";
+import "zitadel/object/v2beta/object.proto";
 
-import "zitadel/resources/webkey/v3alpha/key.proto";
-import "zitadel/resources/object/v3alpha/object.proto";
-import "zitadel/object/v3alpha/object.proto";
-
-option go_package = "github.com/zitadel/zitadel/pkg/grpc/resources/webkey/v3alpha;webkey";
+option go_package = "github.com/zitadel/zitadel/pkg/grpc/webkey/v2beta;webkey";
 
 option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
   info: {
     title: "Web key Service";
-    version: "3.0-preview";
-    description: "This API is intended to manage web keys for a ZITADEL instance, used to sign and validate OIDC tokens. This project is in preview state. It can AND will continue breaking until a stable version is released.";
+    version: "2.0-beta";
+    description: "This API is intended to manage web keys for a ZITADEL instance, used to sign and validate OIDC tokens. This project is in beta state. It can AND will continue breaking until a stable version is released.";
```

Comment:
Not sure if project is correct here
```suggestion
    description: "This API is intended to manage web keys for a ZITADEL instance, used to sign and validate OIDC tokens. This service is in beta state. It can AND will continue breaking until a stable version is released.";
```

Reply:
also added some more documentation